demo/app/location-tracking/add-location-tracking.ts

```diff
@@ -0,0 +1,59 @@
+import Space from '../../../src/Space';
+import { colors } from '../utils/colors';
+
+export const addLocationTracking = (id: string, htmlElement: HTMLElement, space: Space) => {
+  const selectedTracker = space.locations.createTracker((locationChange) => locationChange.currentLocation === id);
+  const unselectedTracker = space.locations.createTracker(
+    (locationChange) => locationChange.previousLocation === id && locationChange.currentLocation !== id,
+  );
+
+  const selectedClasses = ['outline-2', 'outline-dashed'];
+
+  selectedTracker.on((change) => {
+    const self = space.getSelf();
+
+    htmlElement.setAttribute(`data-client-${change.member.clientId}`, 'true');
+
+    if (change.member.clientId === self.clientId) {
+      htmlElement.classList.add(...selectedClasses, 'outline-blue-400');
```

Comment:
Does this mean we're showing our own location, to ourselves? Should't that be just filtered out, like in the members update event?

Reply:
I don't know, I think that would make it more complicated to show where we have selected (need a separate state system to manage that). The different logic for 'self' is just because the color of the avatar is static. I think without showing where we have selected it might be harder to see what's going on. What do you think?

Reply:
I think it makes sense to show the selection in some way to self. For example, even if it is not the profile-coloured border with the name on the top left, we should at least show a black border around the selected element to self — which to others appears with some added CSS props.

Google slides shows a draggable editor component to self:

<img width="1546" alt="Screenshot 2023-04-14 at 12 01 01" src="https://user-images.githubusercontent.com/5900152/232027096-adff8981-8788-4825-9e43-ba88cf8dee7a.png">

Reply:
That makes sense. I wonder though if this creates a bit of discrepancy with `membersUpdate` which does not include self by default (i.e. it would be unexpected to the developer that one behaves differently to the other).

Reply:
Oh - but is that still correct - did we change that behaviour while fixing the bug?

Reply:
Yes, we did actually!